docs/swagger/spec.yml

```diff
@@ -321,6 +476,89 @@ definitions:
         type: string
         description: URL safe version of name
         readOnly: true
+  LayerSource:
```

Comment:
This is a confusing name for this I think since I'd think of a layer's source to be its constituent layer images